scripts/server.sh

```diff
@@ -4,14 +4,6 @@ set -e
 
 echo "Run the RWD Docker container."
 
-if [ -z "$RWD_DATA" ]; then
```

Comment:
Unfortunately, there is a lack of symmetry between `run` and `up` in Docker Compose. If you omit the `-e` flag when using `docker-compose run` it will not be available within the context of the container.